tutorials/baseline/auto_trainer_example.ipynb

```diff
@@ -0,0 +1,571 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "obvious-chicago",
+   "metadata": {},
+   "source": [
+    "# Auto Trainer Tutorial\n",
+    "\n",
+    "Here we want to show you an maximally easy example how to use Knodle out-of-the box. Therefor this tutorial contains\n",
```

Comment:
could you add a little bit more detailed explanation of what autotrainer is? what is the difference between autotrainer and other (denoising specific) trainers?